README.md

```diff
@@ -2,31 +2,37 @@
 
 # crystal-db
 
-Common db api for crystal. You will need to a specific driver to access a database.
+Common DB API for Crystal. You will need a specific driver to access a database.
 
-* [sqlite](https://github.com/crystal-lang/crystal-sqlite3)
-* [mysql](https://github.com/crystal-lang/crystal-mysql)
-* [postgres](https://github.com/will/crystal-pg)
+* [SQLite](https://github.com/crystal-lang/crystal-sqlite3)
+* [MySQL](https://github.com/crystal-lang/crystal-mysql)
+* [PostgreSQL](https://github.com/will/crystal-pg)
 
 ## Installation
 
-Add this to your application's `shard.yml`:
+Add `crystal-db` to your application's `shard.yml`:
 
 ```yaml
 dependencies:
   db:
     github: crystal-lang/crystal-db
 ```
 
+Add the driver for your database, i. e. for Postgres:
+
+```yaml
+dependencies:
+  pg:
+    github: will/crystal-pg
+```
+
 ## Documentation
 
 * [Latest API](http://crystal-lang.github.io/crystal-db/api/latest/)
 * [Crystal book](https://crystal-lang.org/docs/database/)
 
 ## Usage
 
-Since this is an abstract db api, it's usage is through a concrete database driver.
```

Comment:
I guess this was to abstract. What about

> You need to require the specific driver to use also.

So the user is warned to do `require "sqlite3"`

Reply:
What about

> This shard only provides an abstract database API. In order to use it, a specific driver for the intended database has to be required as well: